### ISSUE TITLE
molecule create don't create containers when provisioner.options.tags defined

### ISSUE THREAD

**Issue by popstas** [metacloud/molecule#1038]
# Issue Type

- Bug report

# Molecule and Ansible details

```
ansible 2.4.1.0
molecule 2.4.0
```

# Actual Behaviour

molecule.yml:
``` yaml
---
driver:
  name: docker
platforms:
  - name: molecule-test
    image: ubuntu
    image_version: 16.04
lint:
  name: yamllint
  enabled: False

provisioner:
  name: ansible
  options:
    tags: sometag
```

`molecule create` actually don't creates containers, also, when I created containers and then add `options.tags`, `molecule destroy` don't removes containers.

See example repository - https://github.com/popstas/molecule-tags-create

Travis-CI - https://travis-ci.org/popstas/molecule-tags-create/builds/302513011

I have created role with `molecule init role -r molecule-tags-create`

Example output:

```
$ molecule --debug create

DEBUG: COMMAND
/usr/local/bin/ansible-playbook --tags=micro --inventory=/home/popstas/projects/ansible/viasite-ansible/ansible-role-common/molecule/default/.molecule/ansible_inventory.yml --diff /home/popstas/projects/ansible/viasite-ansible/ansible-role-common/molecule/default/prepare.yml -vvv

    ansible-playbook 2.4.1.0
      config file = /home/popstas/projects/ansible/viasite-ansible/ansible-role-common/molecule/default/.molecule/ansible.cfg
      configured module search path = [u'/usr/local/lib/python2.7/dist-packages/molecule/provisioner/ansible/plugins/libraries', u'/home/popstas/projects/ansible/viasite-ansible/ansible-role-common/library', u'/home/popstas/projects/ansible/viasite-ansible/ansible-role-common/molecule/default/.molecule/library']
      ansible python module location = /usr/local/lib/python2.7/dist-packages/ansible
      executable location = /usr/local/bin/ansible-playbook
      python version = 2.7.12 (default, Nov 19 2016, 06:48:10) [GCC 5.4.0 20160609]
    Using /home/popstas/projects/ansible/viasite-ansible/ansible-role-common/molecule/default/.molecule/ansible.cfg as config file
    Parsed /home/popstas/projects/ansible/viasite-ansible/ansible-role-common/molecule/default/.molecule/ansible_inventory.yml inventory source with yaml plugin

    PLAYBOOK: prepare.yml **********************************************************
    1 plays in /home/popstas/projects/ansible/viasite-ansible/ansible-role-common/molecule/default/prepare.yml

    PLAY [Prepare] *****************************************************************
    META: ran handlers
    META: ran handlers
    META: ran handlers

    PLAY RECAP *********************************************************************
```

**Comment by retr0h** [comment]
@popstas good bug report.  What is happening is Molecule is taking the options passed to the provisioner and using them for create/destroy, which should be excluded.

**Comment by dvigueras** [comment]
Hi, the PR works when doing `molecule create` or `molecule destroy`, but it doesn't work when `create` or `destroy` steps are invoked using `molecule test` command.

**Comment by lobsterdore** [comment]
I can't get this change to work, I have confirmed that I have the updates from #1047  in my venv checkout of Molecule, and yet when I specify tags neither create nor destroy do anything.

**Comment by retr0h** [comment]
@dvigueras @lobsterdore should be corrected in the 2.8 release.
